src/integ_test_resources/common/scripts/device_config_builder.py

```diff
@@ -2,11 +2,17 @@
 
 import json
 import os
+import pathlib
 import sys
 from collections import namedtuple
 
 import boto3
 
+sys.path.append(str(pathlib.Path(__file__).parent.absolute()) + "/..")
+from platforms import Platform
+
+SUPPORTED_PLATFORMS = list(map(lambda p: p.value, Platform))
```

Comment:
Nice! These kinds of functions like map(), filter(), reduce(), and friends are not often used in Python land these days. The more Pythonic way to do it would be:

```python
SUPPORTED_PLATFORMS = [platform.value for platform in Platform]
```

Not that I object to the above syntax or think you should change it. Just a note.

Reply:
I'd prefer to stay idiomatic where I can figure out the idiom. :) Will change in an upcoming PR.